src/pt/apecomics/src/eu/kanade/tachiyomi/extension/pt/apecomics/Capitoons.kt

```diff
@@ -0,0 +1,20 @@
+package eu.kanade.tachiyomi.extension.pt.apecomics
+
+import eu.kanade.tachiyomi.multisrc.mangathemesia.MangaThemesia
+import eu.kanade.tachiyomi.network.interceptor.rateLimit
+import okhttp3.OkHttpClient
+import java.text.SimpleDateFormat
+import java.util.Locale
+
+class Capitoons : MangaThemesia(
+    "Capitoons",
+    "https://capitoons.com",
+    "pt-BR",
+    dateFormat = SimpleDateFormat("dd/MM/yyyy", Locale("pt", "BR")),
```

Comment:
No locale specific patterns, so no need to use a specific locale:
```suggestion
    dateFormat = SimpleDateFormat("dd/MM/yyyy", Locale.ROOT),
```